third_party/move/move-vm/runtime/src/loader.rs

```diff
@@ -610,6 +613,20 @@ pub(crate) struct Loader {
     vm_config: VMConfig,
 }
 
+impl Clone for Loader {
+    fn clone(&self) -> Self {
+        Self {
+            scripts: RwLock::new(self.scripts.read().clone()),
+            module_cache: RwLock::new(self.module_cache.read().clone()),
+            type_cache: RwLock::new(self.type_cache.read().clone()),
+            natives: self.natives.clone(),
+            invalidated: RwLock::new(*self.invalidated.read()),
+            module_cache_hits: RwLock::new(self.module_cache_hits.read().clone()),
+            vm_config: self.vm_config.clone(),
+        }
+    }
+}
```

Comment:
The cloning of the Move VM looks correct. We're not creating any sort of shared stuff here.